ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml

```diff
@@ -71,7 +72,7 @@ items:
           ports:
           - containerPort: 8080
             protocol: TCP
-          resources: {}
+       #  resources: {}
```

Comment:
Why was this commented out compared to original?

Reply:
`resources` appears twice as as key in the same scope (open the whole file; don't think its visible in the snippet). The second one has a value, so I commented this one out .. should verify before merging.

Reply:
These are different. Even though it is empty in both cases and shouldn't make a difference, we also should not be changing something like this - that is up to the workload owner.

`.spec.strategy.resources`
vs
`.spec.template.spec.containers.resources`

Reply:
Are you sure? There's another `resources: ` on line 77 that has limits and requests in it. I think that's the one yamllint says is conflicting.

Reply:
I beg your pardon...you are absolutely right, I missed that one.

In this case, can you just the delete the one with the empty dict? That should resolve this.